source/Irrlicht/CMeshSceneNode.cpp

```diff
@@ -67,7 +67,7 @@ void CMeshSceneNode::OnRegisterSceneNode()
 				if (!mb||(mb->getIndexCount()<1 && !mb->isIndexCountGivenByXFormFeedback()))
                     continue;
 
-				video::IMaterialRenderer* rnd = driver->getMaterialRenderer(mb->getMaterial().MaterialType);
+				video::IMaterialRenderer* rnd = driver->getMaterialRenderer(0);

```

Comment:
the `IMaterialRenderer` class can get nuked.

Reply:
@Crisspl TODO

Reply:
@Crisspl still a TODO

Reply:
resolved nuked long ago